pkg/console/controllers/oauthclients/oauthclients.go

```diff
@@ -116,42 +96,45 @@ func NewOAuthClientsController(
 			routeInformer.Informer(),
 			ingressConfigInformer.Informer(),
 			targetNSsecretsInformer.Informer(),
-			configNSSecretsInformer.Informer(),
-			targetNSDeploymentsInformer.Informer(),
 		).
 		WithFilteredEventsInformers(
 			factory.NamesFilter(api.OAuthClientName),
 			oauthClientSwitchedInformer.Informer(),
 		).
-		WithFilteredEventsInformers(
-			factory.NamesFilter("authentications.config.openshift.io"),
-			crdInformer.Informer(),
-		).
 		WithSyncDegradedOnError(operatorClient).
 		ResyncEvery(wait.Jitter(time.Minute, 1.0)).
 		ToController("OAuthClientsController", recorder.WithComponentSuffix("oauth-clients-controller"))
 }
 
 func (c *oauthClientsController) sync(ctx context.Context, controllerContext factory.SyncContext) error {
-	statusHandler := status.NewStatusHandler(c.operatorClient)
-
 	if shouldSync, err := c.handleManaged(ctx); err != nil {
 		return err
 	} else if !shouldSync {
 		return nil
 	}
 
-	operatorConfig, err := c.consoleOperatorLister.Get(api.ConfigResourceName)
+	statusHandler := status.NewStatusHandler(c.operatorClient)
+
+	authnConfig, err := c.authnLister.Get(api.ConfigResourceName)
 	if err != nil {
 		return err
 	}
 
-	ingressConfig, err := c.ingressConfigLister.Get(api.ConfigResourceName)
+	switch authnConfig.Spec.Type {
+	case "", configv1.AuthenticationTypeIntegratedOAuth:
+	default:
+		// if we're not using integrated oauth, reset all degraded conditions
+		statusHandler.AddConditions(status.HandleProgressingOrDegraded("OAuthClientSecretSync", "", nil))
+		statusHandler.AddConditions(status.HandleProgressingOrDegraded("OAuthClientSync", "", nil))
+		return statusHandler.FlushAndReturn(nil)
+	}
+
+	operatorConfig, err := c.consoleOperatorLister.Get(api.ConfigResourceName)
 	if err != nil {
 		return err
 	}
 
```

Comment:
Add a check to be sure the `authnConfig.Spec.Type` check is AuthenticationTypeOIDC or reset all the conditions you're setting here and return.

Reply:
> Add a check to be sure the `authnConfig.Spec.Type` check is AuthenticationTypeOIDC or reset all the conditions you're setting here and return.

I think I actually assumed this backwards.  https://github.com/openshift/console-operator/pull/861/files?diff=split&w=0#r1483192431 will help